cli/pkg/git/git.go

```diff
@@ -14,6 +16,30 @@ func Clone(repo, dir string, shallow bool) error {
 	return cmd("clone", repo)
 }
 
+func CloneGBM(dir string, pr gh.PullRequest, verbose bool) (*g.Repository, error) {
```

Comment:
I'm not sure about putting something this GBM specific in the `git` package.  Feels like it belongs in `gbm/gbm.go`

Not a blocker but something to consider as a refactor.